### ISSUE TITLE
Cannot link a Deck

### ISSUE THREAD

**Issue by shadowfrillz** [jeremylarner/keyteki#69]
For whatever reason I cannot link 1 of my decks...
Here is the deck:
https://www.keyforgegame.com/deck-details/ea0626b6-4e26-45fc-88a7-fe05cab0efca

**Comment by blinkingline** [comment]
https://www.keyforgegame.com/deck-details/ea0626b6-4e26-45fc-88a7-fe05cab0efca 

That linked right up for me.  Weird.

**Comment by shadowfrillz** [comment]
Yeah... I thought you fixed it because a day later it worked fine...

Feel free to close this out.
